pkg/virt-api/webhooks/arm64.go

```diff
@@ -94,23 +132,34 @@ func setDefaultBootloader(vmi *v1.VirtualMachineInstance) {
 	}
 }
 
-// ValidateVirtualMachineInstanceArm64Setting is validation function for validating-webhook
-func ValidateVirtualMachineInstanceArm64Setting(field *k8sfield.Path, spec *v1.VirtualMachineInstanceSpec) []metav1.StatusCause {
-	var causes []metav1.StatusCause
-	if cause, ok := verifyInvalidSetting(field, spec); !ok {
-		causes = append(causes, cause)
+// setDefaultDisksBus set default Disks Bus, because sata is not supported by qemu-kvm of Arm64
+func setDefaultDisksBus(vmi *v1.VirtualMachineInstance) {
+	if vmi.Spec.Domain.Devices.Disks != nil {
+		for i := range vmi.Spec.Domain.Devices.Disks {
+			disk := &vmi.Spec.Domain.Devices.Disks[i].DiskDevice
+
+			if disk.Disk != nil && (disk.Disk.Bus != "scsi" && disk.Disk.Bus != "virtio") {
```

Comment:
should you check for an empty bus field here instead of excluding overriding specific types? If we get to the modification webhook, that means we already passed the validation webhook and no invalid type should reach this point.